phpunit.xml.dist

```diff
@@ -18,6 +18,7 @@
                Environment variable not found: "APP_SECRET".
         -->
```

Comment:
This TODO comment feel like a work around when defining environment variables here should be the early way to go. Can be removed or modified?

Reply:
Yes! Let's do that in a separate PR. Thanks